src/front/spv/mod.rs

```diff
@@ -424,6 +427,8 @@ pub struct Parser<I> {
     options: Options,
     index_constants: Vec<Handle<crate::Constant>>,
     index_constant_expressions: Vec<Handle<crate::Expression>>,
+    maths_constants: Vec<Handle<crate::Constant>>,
```

Comment:
This is very similar to index constants, in spirit: in both cases we keep a hold of a constant handle, and in both cases there is an associated expression handle. We should probably merge them.

Reply:
We could merge them, but this would make things a lot messier. The `index_constant_expressions` are currently indexed by the index of the vector component that is being extracted. Either we change that by an offset, or we change `DEGREES_TO_RADIANS_CONST_INDEX` and `RADIANS_TO_DEGREES_CONST_INDEX` to go after these 4 index constants. I'd prefer to keep things separate if possible.